stock_location_orderpoint/models/stock_location_orderpoint.py

```diff
@@ -466,33 +466,43 @@ def _prepare_procurements(self, products=False):
     def run_replenishment(self, products=False):
         """Run the replenishment for all potential products or only a selection"""
         procurements = self._prepare_procurements(products)
-        if not procurements:
-            return
-        self.env["procurement.group"].with_context(from_orderpoint=True).run(
-            procurements, raise_user_error=False
-        )
-        self._after_replenishment()
+        if procurements:
+            self.env["procurement.group"].with_context(from_orderpoint=True).run(
+                procurements, raise_user_error=False
+            )
+        return self._after_replenishment()
```

Comment:
I would rather move the logic for getting the current replenishment moves into here and then passing them on downwards, instead of getting them from the result of `_after_replenishment`. 

```suggestion
        replenishment_moves = self._get_current_replenishment_moves()
        self._after_replenishment(replenishment_moves)
        return replenishment_moves
```

Reply:
You are right, this is probably better to put this logic there, I'll make a commit